### PR TITLE
[Will DELETE][duplicate ]leaderboard query and GetPlans_plans_nodes fragment

### PR DESCRIPTION
**Pull request by bighammerw**
## Description

Ticket [SQN-1301](https://onfinality.atlassian.net/browse/SQN-1301?atlOrigin=eyJpIjoiZGJlMGZmMzk0MzNkNDY1ODliZWRjYjk2OWQzOGFhZTgiLCJwIjoiaiJ9)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Changes

- add leaderboard section
- add GetPlans_plans_nodes fragment

### CODE REVIEW

File: packages/network-clients/src/__types__/agreements.generated.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import * as Types from './base-types';
+
```
Comment (lamcc21) [inline packages/network-clients/src/__types__/agreements.generated.ts:2]: generated file shouldn't be committed

**Comment by lamcc21** [comment]
You can also update react-hooks which will use the generated types to create useQuery hook

File: packages/network-query/scripts/codegen.sh [hunk 2]
```diff
@@ -6,6 +6,9 @@ graphql-codegen --require dotenv/config --config types.codegen.ts
 # exchange
 graphql-codegen --require dotenv/config --config exchange.codegen.ts
 
+# leadboard
+graphql-codegen --require dotenv/config --config leaderboard.codegen.ts
```
Comment (CaiYiLiang) [inline packages/network-query/scripts/codegen.sh:10]:
```suggestion
```

File: packages/network-query/leaderboard.codegen.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
+// Copyright 2020-2022 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { CodegenConfig } from '@graphql-codegen/cli';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const config: CodegenConfig = {
```
Comment (CaiYiLiang) [inline packages/network-query/leaderboard.codegen.ts:9]: May I confirm we confirm to include `leaderboard`?

Comment (bighammerw) [reply packages/network-query/leaderboard.codegen.ts:9]: it‘s for fetch topindexer

**Pull request closed without merging**